src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,1138 @@
+                } catch (ResourceNotReachableException e) {
+                    if (e.getMessage().contains(SESSION)) {
+                        // In case it's been rebooted by some other resource - we need to react to that.
+                        // Normally we expect that the authorization request timeouts, since previous one failed with
+                        // a specific error code, so basically we do the same as before - authenticate + retry previous
+                        // request but with the destroy action called first, since init will be done next when
+                        // authentication is requested.
+                        try {
+                            disconnect();
+                            authenticate();
+                            response = execution.execute(request, body);
+                        } catch (Exception ex) {
+                            throw new IOException("Unable to recover the http connection during the request interception: " + ex.getMessage());
+                        }
+                    }
+                } catch (Exception e) {
+                    logger.error("Authentication failed during interception: " + e.getMessage());
+                }
+            }
+            return response;
+        }
+    }
+
+    private static final String SESSION = "rest/current/session";
+    private static final String STATUS = "rest/system/status";
+    private static final String CONFERENCING_CAPABILITIES = "rest/conferences/capabilities";
+    private static final String CONFERENCES = "rest/conferences"; // POST for calling a single participant, GET to list all
+    private static final String CONFERENCE = "rest/conferences/%s"; // DELETE to disconnect
+    private static final String MEDIASTATS = "rest/conferences/%s/mediastats";
+    private static final String SHARED_MEDIASTATS = "/rest/mediastats";
+    private static final String AUDIO = "rest/audio";
+    private static final String AUDIO_MUTED = "rest/audio/muted";
+    private static final String VIDEO_MUTE = "rest/video/local/mute";
+    private static final String CONTENT_STATUS = "rest/cameras/contentstatus";
+    private static final String VOLUME = "rest/audio/volume";
+    private static final String SYSTEM = "rest/system";
+    private static final String REBOOT = "rest/system/reboot";
+    private static final String COLLABORATION = "rest/collaboration";
+    private static final String MICROPHONES = "rest/audio/microphones";
+    private static final String APPS = "rest/system/apps";
+    private static final String SESSIONS = "rest/current/session/sessions";
+
+    private static final String CONTROL_MUTE_VIDEO = "Mute Local Video";
+    private static final String CONTROL_MUTE_MICROPHONES = "Mute Microphones";
+    private static final String CONTROL_AUDIO_VOLUME = "Audio Volume";
+    private static final String CONTROL_REBOOT = "Reboot";
+
+    private static int DEFAULT_CALL_RATE;

```

Comment:
When used as a property, cannot not be static, static means the same value for all instances of adapter. Property is technically "instance property".